docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -239,7 +243,7 @@ To gather logging and metric data from your swarm, first [enable logging for the
 
 Docker EE for IBM Cloud employs a flexible architecture and integration with IBM Cloud that you can use to leverage IBM Cloud resources and customize your swarm environment. Docker EE UCP exposes the standard Docker API, and as such, includes certain functions that instead should be done by using Docker EE for IBM Cloud capabilities.
 
-> Self-healing capabilities so you don't need to modify cluster infrastructure.
+> Self-healing capabilities so you don't have to modify cluster infrastructure.
```

Comment:
Again, this is clobbering editorial changes I made on purpose across the docset.

Reply:
Sorry about that; I think when I rebased I missed these changes :'(